### DISCUSSION TITLE
How can we change the colours of axis titles in wave plots

### DISCUSSION THREAD

**Discussion by VijithaEkanayake-zz** [h2oai/wave#829] [Q&A]
In my wave application, all the plots are generated using the wave plots API. However, when the application switched the neon theme axis titles are not clearly visible due to fact that the font color and the plot color are somewhat identical.  How can we change the colors of axis titles in wave plots? It would be nice if we could control the font size of the titles as well.

<img width="1457" alt="Screenshot 2021-05-21 at 09 45 06" src="https://user-images.githubusercontent.com/5285626/119085645-bd723e00-ba21-11eb-8e3c-973431c1814c.png">

**Comment by naeemanaz** [comment]
Yes

**Comment by vopani** [comment]
Its tracked here https://github.com/h2oai/wave/issues/664
